app/screens/Resources.js

```diff
@@ -29,31 +29,31 @@ const Resources = () => {
                 </Text>
                 <View style={s.hr}></View>
                 <Pressable onPress={() => Linking.openURL('https://matchplay.events')} style={[s.logoWrapper,{backgroundColor:'#f5f5ff',paddingVertical: 10,}]}>
-                    <Image source={require('../assets/images/Resource_Matchplay.png')} width={theme.theme === 'dark' ? deviceWidth - 70 : deviceWidth - 50} />
+                    <Image source={require('../assets/images/Resource_Matchplay.png')} style={{width:deviceWidth-70,height:(deviceWidth-70)/8.08,resizeMode: 'stretch'}} />
                 </Pressable>
                 <Text style={s.text}>
                     <Text style={s.textLink} onPress={() => Linking.openURL('https://matchplay.events/')}>{`Match Play Events`}</Text>
                     {` is a tournament app which makes it easy to organize tournaments on any device. Your players can follow standings and results live on their own mobile devices.`}
                 </Text>
                 <View style={s.hr}></View>
                 <Pressable onPress={() => Linking.openURL('https://pindigo.app')} style={[s.logoWrapper,{backgroundColor:'#363377',        paddingVertical: 10,}]}>
-                    <Image source={require('../assets/images/Resource_Pindigo.png')} resizeMode="contain" width={deviceWidth - 70} />
+                    <Image source={require('../assets/images/Resource_Pindigo.png')} style={{width:deviceWidth-70,height:(deviceWidth-70)/3.53,resizeMode: 'stretch'}} />
                 </Pressable>
                 <Text style={s.text}>
                     <Text style={s.textLink} onPress={() => Linking.openURL('https://pindigo.app')}>{`Pindigo`}</Text>
                     {` is an app for recording your scores. You can track all your high scores and compare them with friends.`}
                 </Text>
                 <View style={s.hr}></View>
                 <Pressable onPress={() => Linking.openURL('https://www.pinballnews.com/site/')} style={[s.logoWrapper]}>
-                    <Image source={require('../assets/images/Resource_PinballNews.png')} resizeMode="contain" width={deviceWidth - 50} />
+                    <Image source={require('../assets/images/Resource_PinballNews.png')} style={{width:deviceWidth-50,height:(deviceWidth-50)/4.07,resizeMode: 'stretch'}} />
```

Comment:
Looks like it was a chore to determine the heights of these images! 😅

Reply:
Totally bonkers that image scaling isn’t better handled in RN.

Maybe there’s an opportunity to create a new package that handles this 😜 The old one isn’t keeping pace https://github.com/ihor/react-native-scalable-image
Or have this functionality added to RN itself!!